codec/src/main/java/io/netty/handler/codec/ByteToMessageDecoder.java

```diff
@@ -200,12 +236,12 @@ public void setCumulator(Cumulator cumulator) {
     }
 
     /**
-     * Set the number of reads after which {@link ByteBuf#discardSomeReadBytes()} are called and so free up memory.
-     * The default is {@code 16}.
+     * @deprecated discards are now performed based on accumulated size of read bytes
```

Comment:
- discards -> Discards
- missing period at the end of the sentence.

How about adding a one-line description so that the generated Javadoc does not show an empty description cell in the summary table? e.g.

```java
/**
 * This method does nothing. Discards are now performed based on accumulated size of read bytes.
 * @deprecated This method does nothing.
 */
```